index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="index.css" rel="stylesheet" type="text/css">
+    <style type="text/css"></style>
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="grid-wrapper">
+
+        <header class="masthead">
+            <nav class="toolbar">
+                <ul>
+                    <li style="padding-top: 0.5rem;"><a href="https://www.facebook.com/events/1455321544773645/"><img
+                                src="statics/images/facebook.png" alt="Facebook"></a></li>
+                    <li style="padding-top: 0.5rem;"><a href="https://twitter.com/cckorea"><img
+                                src="statics/images/twitter.png" alt="Twitter"></a>
+                    </li>
+            <li><a href="https://summit.creativecommons.org/" target="_blank">Logout</a></li>
+            <li><a href="https://summit.creativecommons.org/" target="_blank">Mypage</a></li>
```

Comment:
- [optional] you can add a home link in your mobile menu to ease accessibility just in case a user doesnt know the heading is the home link
![Screenshot (736)](https://user-images.githubusercontent.com/65298048/211082604-d54afbf6-cfe4-45e5-be84-ea2a936af917.png)

Reply:
added. Thanks.

<img width="413" alt="image" src="https://user-images.githubusercontent.com/39811497/211100965-24eae092-489d-45be-b68d-ce6be205feab.png">